apps/web/src/components/(dashboard)/common/command-menu.tsx

```diff
@@ -53,6 +55,34 @@ export function CommandMenu({ open, onOpenChange }: CommandMenuProps) {
   const [search, setSearch] = useState('');
   const [pages, setPages] = useState<string[]>([]);
 
+  const [searchResults, setSearchResults] = useState<
+    { label: string; path: string; recipents: string[] }[]
+  >([]);
+
+  const { isLoading: isSearchingDocuments, refetch: findDocuments } =
```

Comment:
Why not use the `data` property instead? Using onSuccess on queries will be deprecated with React Query 5 afaik

Reply:
makes sense, I'll make change it